SimpleBrowser/Internal/StringExtensions.cs

```diff
@@ -90,8 +90,15 @@ public static Color ToColor(this string s)
 		failed:
 			if(char.IsDigit(s[0]) || s[0] == '-')
 				return Color.FromArgb(s.ToInt());
-			return Color.FromKnownColor(s.ToEnum<KnownColor>());
-		}
+
+        // In .NET 4 the Color.FromName method also exists, we could use it.
+        // The string parameter matches with the same names as in the KnownColor enumeration (see docs).
+#if NET40
+            return Color.FromKnownColor(s.ToEnum<KnownColor>());
+#else
+            return Color.FromName(s);
+#endif
+        }
```

Comment:
@kevingy 
Can you look at this? To avoid code branching, we could use "Color.FromName", as per documentation this should match to the names of KnownColor (you try to parse the string to this enum). Docs : https://docs.microsoft.com/en-us/dotnet/api/system.drawing.color.fromname?view=netframework-4.7.1#System_Drawing_Color_FromName_System_String_

Reply:
@Styxxy 
Perhaps this is a case where we can cure the disease by killing the patient. This method isn't referenced. The entire method can be deleted, eliminating the code branch, and everything works perfectly.

It's curious that a headless browser (that do any rendering) has methods that reference colors. If I had a guess, @axefrog needed one of these methods and brought in the entire class, thinking that the methods might one day be needed.

Reply:
@kevingy 
Good catch! I went  through the string extension methods and removed also a bunch of others (that are not used).